batch/batch/front_end/front_end.py

```diff
@@ -219,6 +219,18 @@ def cast_query_param_to_bool(param: Optional[str]) -> bool:
     return True
 
 
+def deprecated(fun):
```

Comment:
This is a much nicer solution than I proposed. Thank you!

Reply:
Thanks! Sorry, meant to reply to your comment after the tests ran but forgot